fcl/dunefdvd/reco/workflow_reco_dunevd10kt.fcl

```diff
@@ -66,29 +90,57 @@ dunefd_vertdrift_tpc:
     cvneval,
     energyrecnumu,
     energyrecnue,
-    energyrecnc
+    energyrecnue
```

Comment:
energyrecnue twice?

Reply:
Good catch!
Corrected to be energyrecnc